models/tokens/tokens_erc20.sql

```diff
@@ -24,4 +24,5 @@ UNION ALL
 SELECT 'fantom' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_fantom_erc20') }}
 UNION ALL
 SELECT 'base' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_base_erc20') }}
-
+UNION ALL
+SELECT 'celo' as blockchain, contract_address, symbol, decimals FROM {{ ref('tokens_celo_erc20') }}
```

Comment:
Hi @Stakeridoo,
we may need to make sure the table `tokens_celo_erc20` existed first.

Reply:
yes, this will need built first. feel free to initiate it, if not already in main branch 🙏